doc/classes/TextEdit.xml

```diff
@@ -764,7 +764,8 @@
 			<return type="bool" />
 			<param index="0" name="caret_index" type="int" default="0" />
 			<description>
-				Returns [code]true[/code] if the caret is visible on the screen.
+				Returns [code]true[/code] if the caret is visible, [code]false[/code] otherwise. A caret will be considered hidden if it is outside the scrollable area when scrolling is enabled.
```

Comment:
```suggestion
				Returns [code]true[/code] if the caret is visible. A caret will be considered hidden if it is outside the scrollable area when scrolling is enabled.
```
I think explicit "false otherwise" is unnecessary.